src/train.py

```diff
@@ -97,36 +97,100 @@ def speech_loss(gold_mel, stop_label, pred_mel, stop_pred):
     stop_loss = F.binary_cross_entropy_with_logits(stop_pred.squeeze(), stop_label)
     return pred_loss + stop_loss
 
+
+def cross_entropy(input, target, size_average=True):
```

Comment:
Why not just use the torch cross entropy here? Is it the smoothing term?  Why is the smoothing term important here? (I didn't read the original paper that closely...)

Reply:
Yes, torch loss does not allow for soft targets. The smoothing term has been shown to give better convergence on classification problems and was used in the Almost Unsupervised paper.